tests/InputFiles/ScalarWave/Input1DPeriodic.yaml

```diff
@@ -0,0 +1,30 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+# Executable: EvolveScalarWave1D
```

Comment:
I don't feel particularly strongly about this, but I like the idea of there being a newline after the licence. I find it makes it easier to parse since I've gotten into the habit of ignoring the licence text.